test/parallel/test-webcrypto-encrypt-decrypt.js

```diff
@@ -103,11 +105,11 @@ const { subtle, getRandomValues } = require('crypto').webcrypto;
       length: 256
     }, true, ['encrypt', 'decrypt']);
 
-    const ciphertext = await subtle.encrypt({
-      name: 'AES-GCM', iv }, key, buf);
+    const ciphertext = await subtle.encrypt({ name: 'AES-GCM', iv }, key, buf);
```

Comment:
```suggestion
    const ciphertext = await subtle.encrypt(
      { name: 'AES-GCM', iv }, key, buf,
    );
```

Reply:
I'm not sure why this one should be broken out into more than one line. Maybe that's a bit easier to read?

Reply:
I'm not sure this one should be broken up. Is it significantly easier to read that way?

Reply:
Oh, consistency with the subsequent assignment seems like a solid reason.